### PR TITLE
Register policies by framework

### PR DESCRIPTION
**Pull request by kartikp10**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

Changes in this PR:

1. "frameworks" are now returned in the runConfig response. Therefore, custom cloned checks are appended to the respective registries based on frameworks. This fixes an issue where a check with multiple frameworks and resource_type = all was only appended to the TF registry. 
2. only for cloned platform custom policies  with resource_type == all, the `checkov --list` command will now render the checks in the output as shown below

```
| 2063 | kpande_aws_1649106703226 | resource                         | all                                                                          | Terraform Compliance Check tags (!!!TEST!!!)                                                                                                                                                             | Terraform               |
| 2064 | kpande_aws_1649106703226 | resource                         | all                                                                          | Terraform Compliance Check tags (!!!TEST!!!)                                                                                                                                                             | CloudFormation          |
| 2065 | kpande_aws_1649172872913 | resource                         | all                                                                          | kptest                                                                                                                                                                                                   | CloudFormation          |
| 2066 | kpande_aws_1649179097048 | resource                         | all                                                                          | kptest2                                                                                                                                                                                                  | CloudFormation          |

```

### CODE REVIEW

File: checkov/common/util/docs_generator.py [hunk 1]
```diff
@@ -75,6 +78,9 @@ def add_from_repository(registry: Union[BaseCheckRegistry, BaseGraphRegistry], c
         graph_registry.load_checks()
         add_from_repository(graph_registry, "resource", "Terraform")
     if any(x in framework_list for x in ("all", "cloudformation")):
+        graph_registry = get_graph_checks_registry("cloudformation")
```
Comment (mikeurbanski1) [inline checkov/common/util/docs_generator.py:81]: Why is this part necessary only for CFN?

Comment (gruebel) [reply checkov/common/util/docs_generator.py:81]: I think we should add it to Kubernetes and Bicep too

**Review by mikeurbanski1 — APPROVED** [review]
